e2e/scripts/utils.ts

```diff
@@ -1,17 +1,17 @@
 import { Address, createPublicClient, erc20Abi, http } from 'viem'
-import { celoAlfajores } from 'viem/chains'
+import { celo } from 'viem/chains'
 import { REFILL_TOKENS } from './consts'
 
 export async function checkBalance(
   address: Address,
-  minBalance = 10,
+  minBalance = 1,
```

Comment:
Will update when the e2e faucet has more funds: [Slack request](https://valora-app.slack.com/archives/C05BHD5AGSW/p1731717707997379).

Reply:
Updated in https://github.com/valora-inc/wallet/pull/6229/commits/ba3c3ec2b9e40a01ff402cb0d1dab6716d5bc350!